config/packages/test/content_api.yaml

```diff
@@ -0,0 +1,15 @@
+content_api:
+    services:
+        articles:
+            items: libero.dummy_api_test.content_api.articles
+
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    libero.dummy_api_test.content_api.articles:
+        class: Libero\ContentApiBundle\Adapter\FilesystemItems
+        arguments:
+            - '%env(ARTICLES_PATH)%'
```

Comment:
wouldn't use an environment variable in general but it's for testing so :+1:

Reply:
I'd rather 'mount' this file from the tests, but currently no way to do it.